_themes/rtd-blog/social.html

```diff
@@ -1,19 +1,53 @@
 
-<!-- Begin MailChimp Signup Form -->
-<h3> Subscribe to our mailing list </h3>
-<p>
-Sign up for our blog and we'll send you information about Sphinx and Read the Docs on a regular basis.
-</p>
-<div id="mc_embed_signup">
-<form action="//readthedocs.us3.list-manage.com/subscribe/post?u=a6a22369cc2b356379cf789ca&amp;id=a85a83a5a5" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-    <div id="mc_embed_signup_scroll">
-    <input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" required>
-    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_a6a22369cc2b356379cf789ca_a85a83a5a5" tabindex="-1" value=""></div>
-    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+<!-- Begin MailerLite Signup Form -->
+  <div id="mlb2-4710274" class="ml-form-embedContainer ml-subscribe-form ml-subscribe-form-4710274">
+    <div class="ml-form-align-center">
+      <div class="ml-form-embedWrapper embedForm">
+        <div class="ml-form-embedBody ml-form-embedBodyDefault row-form">
+          <div class="ml-form-embedContent" style="">
+            <h3 style="margin-left: 0">Subscribe to our mailing list</h3>
+            <p>Sign up for our blog and we'll send you information about Sphinx and Read the Docs on a regular basis.</p>
+          </div>
+          <form class="ml-block-form" action="https://static.mailerlite.com/webforms/submit/p8b7z2" data-code="p8b7z2" method="post" target="_blank">
+            <div class="ml-form-formContent">
+              <div class="ml-form-fieldRow ml-last-item">
+                <div class="ml-field-group ml-field-email ml-validate-email ml-validate-required">
+                  <input aria-label="email" aria-required="true" type="email" class="form-control" data-inputmask="" name="fields[email]" placeholder="email address" autocomplete="email" style="width: 90%">
+                </div>
+              </div>
+            </div>
+            <div class="ml-form-checkboxRow ml-validate-required">
+              <label class="checkbox"> <input type="checkbox"> <div class="label-description"> <p></p> <p>I wish to subscribe to the Read the Docs newsletter to receive news and updates. I&nbsp;know I&nbsp;can unsubscribe at any time.</p> <p></p> </div> </label>
```

Comment:
What is the goal here? I don't think this really needs to be a checklist, but probably copy under the subscribe link?

Reply:
Simplified this a bit

Reply:
IIRC, this is required by GDPR. Just copy is not enough. After a quick google search, many sites mention something like

> Consent requires a positive opt-in. Don't use pre-ticked boxes or any other method of default consent.

Also, I remembered that we had to add this checkbox to our PyCamp web page.

Reply:
If it's required for GDPR, we should do an opt-in email after they have subscribed. A small button on the page is not going to meaningfully be an opt-in (since it can be done by anyone on the web page).